src/components/navBar.js

```diff
@@ -0,0 +1,22 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import { NavLink, Outlet } from 'react-router-dom';
+import './Navbar.css';
+
+export default function NavBar() {
+  return (
+    <div>
+      <header>
+        <nav>
+          <h1>Bookstore CMS</h1>
+          <div>
+            <NavLink to="/">Books</NavLink>
+            <NavLink to="#">Category</NavLink>
+          </div>
+        </nav>
+      </header>
+      <main className="navMenu">
+        <Outlet />
+      </main>
+    </div>
+  );
+}
```

Comment:
- [ ] Your navigation links do not work. I am unable to navigate to the category page. Kindly set up the correct routing in the App component.